portfolio_scraper.py

```diff
@@ -9,8 +9,8 @@
 			platform.login()
 			total_account_value = str(platform.get_account_value()).replace('.', ',')
 			available_funds = str(platform.get_available_funds()).replace('.', ',')
-			print('{} - Total Account Value:	{} {}'.format(platform_name, total_account_value, 'kr' if isinstance(platform, Nordnet) else '€'))
-			print('{} -    Available Founds:	{} {}'.format(platform_name, available_funds, 'kr' if isinstance(platform, Nordnet) else '€'))
```

Comment:
Jeg tænker at dette script er noget der kører i baggrunden og derfor har jeg også fjernet de to prints i min lokale version.